code/game/objects/items/weapons/kitchen.dm

```diff
@@ -70,6 +70,20 @@
 	desc = "It's a fork. Sure is pointy."
 	icon_state = "fork"
 
+/obj/item/kitchen/utensil/fork/suicide_act(mob/user)
+	user.visible_message("<span class='suicide'>[user] is putting the [src] into the garbage disposal! It looks like [user.p_theyre()] trying to commit suicide!</span>")
+	var/message = "DING DING DING DING DING DING DING DING DING"
+	var/span = "reallybig"
```

Comment:
i would just put this directly into the audible message string

Reply:
Done